### PR TITLE
OCP 4.11 - installation doc updates from IBM P

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s): 4.11

Issue:
https://issues.redhat.com/browse/MULTIARCH-2482

Link to docs preview:
http://file.del.redhat.com/~aprabhu/411_upi_install/
http://file.del.redhat.com/~aprabhu/411_upi_install/installing/installing_ibm_power/installing-ibm-power.html#next-steps
http://file.del.redhat.com/~aprabhu/411_upi_install/installing/installing_ibm_power/installing-restricted-networks-ibm-power.html#next-steps

Additional information:
Draft : https://docs.google.com/document/d/1244_6D86D2UJPp6sNc2iNp-Jfh0RL93pDf8f310A00Q/edit#

### CODE REVIEW

File: installing/installing_ibm_power/installing-ibm-power.adoc [hunk 1]
```diff
@@ -123,3 +123,4 @@ include::modules/cluster-telemetry.adoc[leveloffset=+1]
 * xref:../../post_installation_configuration/cluster-tasks.adoc#available_cluster_customizations[Customize your cluster].
 * If necessary, you can
 xref:../../support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc#opting-out-remote-health-reporting_opting-out-remote-health-reporting[opt out of remote health reporting].
+* The link:https://github.com/ocp-power-automation/rsct-operator[RSCT Operator] can be installed to enable RSCT on the cluster.
```
Comment (mjturek) [inline installing/installing_ibm_power/installing-ibm-power.adoc:126]: can you add a link to the README at the end of the sentence? Something like "...to enable RSCT on the cluster. Installation instructions are provided link:https://github.com/ocp-power-automation/rsct-operator/blob/main/README.md[here]

File: installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc [hunk 2]
```diff
@@ -127,3 +127,4 @@ include::modules/installation-complete-user-infra.adoc[leveloffset=+1]
 * xref:../../post_installation_configuration/machine-configuration-tasks.adoc#rhcos-enabling-multipath_post-install-machine-configuration-tasks[Enabling multipathing with kernel arguments on {op-system}].
 * xref:../../post_installation_configuration/cluster-tasks.adoc#available_cluster_customizations[Customize your cluster].
 * If the mirror registry that you used to install your cluster has a trusted CA, add it to the cluster by xref:../../openshift_images/image-configuration.adoc#images-configuration-cas_image-configuration[configuring additional trust stores].
+* The link:https://github.com/ocp-power-automation/rsct-operator[RSCT Operator] can be installed to enable RSCT on the cluster.
```
Comment (mjturek) [inline installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc:130]: ditto

**Review by mjturek — COMMENTED** [review]
Small changes requested

File: installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc (hunk 2 above)
Comment (mjturek) [reply installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc:130]: I'm fine with this as is, but just something to consider if you wind up pushing another revision

File: installing/installing_ibm_power/installing-ibm-power.adoc (hunk 1 above)
Comment (mjturek) [reply installing/installing_ibm_power/installing-ibm-power.adoc:126]: I'm fine with this as is, but just something to consider if you wind up pushing another revision

Comment (alishaIBM) [reply installing/installing_ibm_power/installing-ibm-power.adoc]: Thanks Mike. Above change has been incorporated into the PR.

**Comment by GroceryBoyJr** [comment]
More useful preview links would be:
http://file.del.redhat.com/~aprabhu/411_upi_install/installing/installing_ibm_power/installing-ibm-power.html#next-steps

**Review by sferich888 — CHANGES_REQUESTED** [review]
We should remove all github references to https://github.com/ocp-power-automation/rsct-operator#installing-the-rsct-operator

File: installing/installing_ibm_power/installing-ibm-power.adoc [hunk 3]
```diff
@@ -123,3 +123,4 @@ include::modules/cluster-telemetry.adoc[leveloffset=+1]
 * xref:../../post_installation_configuration/cluster-tasks.adoc#available_cluster_customizations[Customize your cluster].
 * If necessary, you can
 xref:../../support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc#opting-out-remote-health-reporting_opting-out-remote-health-reporting[opt out of remote health reporting].
+* The RSCT Operator can be installed to enable RSCT on the cluster. Installation instructions are provided link:https://github.com/ocp-power-automation/rsct-operator[here].
```
[the remark this answers is not in the document]
Comment (GroceryBoyJr) [reply installing/installing_ibm_power/installing-ibm-power.adoc:126]: @aprabhu Please remove this update. Referring customers to installing the RSCT Operator from a github repo is not supported by Red Hat and has been vetoed by SME review.

File: installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc [hunk 4]
```diff
@@ -127,3 +127,4 @@ include::modules/installation-complete-user-infra.adoc[leveloffset=+1]
 * xref:../../post_installation_configuration/machine-configuration-tasks.adoc#rhcos-enabling-multipath_post-install-machine-configuration-tasks[Enabling multipathing with kernel arguments on {op-system}].
 * xref:../../post_installation_configuration/cluster-tasks.adoc#available_cluster_customizations[Customize your cluster].
 * If the mirror registry that you used to install your cluster has a trusted CA, add it to the cluster by xref:../../openshift_images/image-configuration.adoc#images-configuration-cas_image-configuration[configuring additional trust stores].
+* The RSCT Operator can be installed to enable RSCT on the cluster. Installation instructions are provided link:https://github.com/ocp-power-automation/rsct-operator[here].
```
[the remark this answers is not in the document]
Comment (GroceryBoyJr) [reply installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc:130]: @aprabhu Please remove this update. Referring customers to installing the RSCT Operator from a github repo is not supported by Red Hat and has been vetoed by SME review.

**Review by GroceryBoyJr — COMMENTED** [review]
@alishaIBM Please see review feedback provided in the PR.

File: installing/installing_ibm_power/installing-ibm-power.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (alishaIBM) [reply installing/installing_ibm_power/installing-ibm-power.adoc]: Acknowledge.

File: installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (alishaIBM) [reply installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc]: Acknowledge.

File: installing/installing_ibm_power/installing-ibm-power.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (alishaIBM) [reply installing/installing_ibm_power/installing-ibm-power.adoc]: FYI : I have removed the RSCT Operator reference from both the assemblies.

**Review by GroceryBoyJr — COMMENTED** [review]
/LGTM - Please stand by for merge review.

**Pull request merged**